steps/dropbox-upload-v2/0.0.1/step.yml

```diff
@@ -0,0 +1,62 @@
+title: dropbox-upload-v2
+summary: |
+  upload a file to dropbox and get a shareable link in return
+description: |
+  using this step you can send downloadable links via any platform
+website: https://github.com/AlexPinhasov/bitrise-step-dropbox-upload-v2
+source_code_url: https://github.com/AlexPinhasov/bitrise-step-dropbox-upload-v2
+support_url: https://github.com/AlexPinhasov/bitrise-step-dropbox-upload-v2/issues
+published_at: 2021-08-23T11:22:49.406953+03:00
+source:
+  git: https://github.com/AlexPinhasov/bitrise-step-dropbox-upload-v2.git
+  commit: 328cf9a232efcf656bd62fdf3bfd692c733b7f88
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- utility
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
+  - name: wget
+  apt_get:
+  - name: git
+  - name: wget
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- dropbox_secret: null
+  opts:
+    description: |
+      You have to generate a secret access token in [the developer website](https://www.dropbox.com/developers).
+      Create an application and go in the setting's application.
+    is_required: true
```

Comment:
Should be sensitive

Reply:
I didn't understand the comment

Reply:
Look at the docs: https://devcenter.bitrise.io/contributors/create-your-own-step/#sensitive-inputs

Reply:
Got it, added.